cli/cli.go

```diff
@@ -53,6 +55,35 @@ func (c *CLI) ReadEnv() {
 	}
 }
 
+const ContextFileName = ".hcloud-context"
+
+func (c *CLI) ReadContextFile() {
+	if c.Config == nil {
+		return
+	}
+
+	data, err := ioutil.ReadFile(ContextFileName)
+	if os.IsNotExist(err) {
+		return
+	}
+	if err != nil {
+		log.Printf("warning: failed to read %s file: %s\n", ContextFileName, err)
+		return
+	}
+
+	name := string(bytes.TrimSpace(data))
+	if name == "" {
+		log.Printf("warning: ignoring invalid context in %s\n", ContextFileName)
+		return
+	}
+	if context := c.Config.ContextByName(name); context != nil {
+		c.Config.ActiveContext = context
+		c.Token = context.Token
```

Comment:
some output would be good, showing something like `temporary switching to context your-context`. if called from terraform or any automation script it would be helpful to have such output in the logs. could be hard to debug being in an auto-context and not knowing it.

also some kind of control for this output would be good maybe a `--verbose` option to show it or `--quiet` to supress it.

the output is currently like that if called from a lua script (i call it twice) for showing the context in the command prompt:
```
hcloud: warning: context "not-existing-context" specified in .hcloud-context does not exist
hcloud: warning: context "not-existing-context" specified in .hcloud-context does not exist
C:\terraform\project (development) (hcc: c33s-development)(tfws: development)
```
so being able to call it with `--quiet` from my lua script would be cool.